aten/src/ATen/function_wrapper.py

```diff
@@ -179,7 +179,7 @@ def TypedDict(name, attrs, total=True):  # type: ignore
 STATIC_DISPATCH_FUNCTION_SWITCH_BODY = CodeTemplate("""\
 at::AutoNonVariableTypeMode _var_guard(true);
 switch(dispatchKeyToBackend(c10::impl::dispatchTypeId(${key_set},
-                            c10::DispatchKeySet(c10::DispatchKeySet::FULL).remove(DispatchKey::BackendSelect)))) {
+                            c10::DispatchKeySet(c10::DispatchKeySet::FULL).remove(DispatchKey::BackendSelect).remove(DispatchKey::Profiler)))) {
```

Comment:
@ezyang I added this because it sounded like what you were referring to on the VC, but it looks like the mobile tests are still failing in `dispatchKeyToBackend`. I guess I'll try to get to debugging this

Reply:
Yeah, removal here is what I would have expected would work. Odd.